xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.h

```diff
@@ -136,17 +142,17 @@ namespace OVERLAY {
     void SetSubtitleVerticalPosition(const int value, bool save);
 
     /*!
-     * \brief Reset the subtitle position to default value
+     * \brief Get the subtitle vertical margin in percentage
+     * \return The percentage value
      */
-    void ResetSubtitlePosition();
+    static double GetSubtitleVerticalMarginPerc();
```

Comment:
Any reason for a double instead of float? I don't think we need that level of precision here (also double is 64bit)

Reply:
because i kept same type of `GetNumber` return type
ofc i can change it and add a type cast

Reply:
done